src/geos_functions.jl

```diff
@@ -941,7 +941,7 @@ getGeometries(ptr::GEOSGeom, context::GEOSContext = _context) =
 
 # Converts Geometry to normal form (or canonical form).
 # Return -1 on exception, 0 otherwise.
-function Base.normalize!(ptr::GEOSGeom, context::GEOSContext = _context)
+function LinearAlgebra.normalize!(ptr::GEOSGeom, context::GEOSContext = _context)
```

Comment:
This should now be `function normalize!(ptr::GEOSGeom, context::GEOSContext = _context)`

We shouldn't need the dependency on LinearAlgebra.